mercury-framework/examples/README.md

```diff
@@ -30,9 +30,26 @@ The customer credit rating service domain always returns the same value for any
 1. Create a Minikube cluster with the Mercury Operator and all the dependencies. 
 Look at the [Mercury Operator installation instructions](../../mercury-operator/README.md)
 2. Create a Service Domain Cluster
+
+   `minikube start -p service-domain`
+
 3. Create the different Service Domains (Customer Offer, Party Routing Profile and Customer
 Credit Rating)
+
+   `kubectl apply -f deploy/01_deployment.yaml`
```

Comment:
This resource should be replaced with the custom resources that the operator will use. ServiceDomainCluster and ServiceDomains (one for each)